__tests__/app.test.js

```diff
@@ -304,3 +304,20 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  describe("Functionality", () => {
+    test("200: should return an array of objects with keys of username, name and avatar_url", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.users)).toBe(true);
```

Comment:
as we've used a forEach here, we dont need to check whether it is an array as the forEach working implies that it is an array